src/client-scopes/add/components/ScriptComponent.tsx

```diff
@@ -20,7 +20,20 @@ export const ScriptComponent = ({
     <FormGroup
       label={t(label!)}
       labelIcon={
-        <HelpItem helpText={t(helpText!)} forLabel={t(label!)} forID={name!} />
+        <HelpItem
+          helpText={
+            <span>
+              {helpText?.split("\n").map((item, key) => (
+                <Fragment key={key}>
+                  {item}
+                  <br />
+                </Fragment>
+              ))}
+            </span>
+          }
```

Comment:
If you are looking to preserve the white-space, perhaps you can wrap this in a `pre` tag instead?
```suggestion
          helpText={<pre>{t(helpText!)}</pre>}
```

Reply:
a `<pre` will use another font and mess up the layout:
![image](https://user-images.githubusercontent.com/51133/134869530-ce6950e6-17e9-44ef-a7f4-cb1327266660.png)

Reply:
Ahhh yes. We can use some styling here instead by adding `white-space: pre-wrap;` to the containing element.